nautobot/core/tests/test_csv.py

```diff
@@ -269,27 +269,29 @@ def test_round_trip_export_import(self):
         self.assertEqual(response.status_code, 200)
         # uploading the CSV always returns a 200 code with a page with an error message on it
         # ensure we don't have that error message
-        self.assertNotIn("FORM-ERROR", response.content.decode(response.charset))
-        self.assertEqual(Device.objects.count(), 4)
+        # nautobot/nautobot#1543 - assert entry did not get updated
+        self.assertIn("FORM-ERROR", response.content.decode(response.charset))
+        # since the same IDs are used the existing records were updated instead of created new
```

Comment:
Taking my first look at this in a while, I wonder if it'd be more in the spirit of what this test was originally checking for if we instead (up at line 265) were to replace the `id` columns in the data with new values as well and keep the below steps the same as they were? What do you think?

Reply:
Yeah I think you're right.